fastn-core/src/commands/serve.rs

```diff
@@ -186,6 +186,8 @@ pub async fn serve(
         return Ok(clear_sid2(&req));
     }
 
+    handle_migrations(config).await?;
```

Comment:
On a second thought, lets not call this function here, we can apply migration when Config object is first constructed. Since we re-use config across requests this will be more optimal.

Reply:
done